src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -6,46 +6,69 @@ import { Component } from 'react';
 import { connect } from 'react-redux';
 import { shell, remote } from 'electron';
 
+import packageJson from '../../../package.json';
+
 import {
   createNewProjectStart,
   showImportExistingProjectPrompt,
+  clearConsole,
 } from '../../actions';
+import { isMac } from '../../services/platform.service';
+import { getSelectedProject } from '../../reducers/projects.reducer';
+import { getDevServerTaskForProjectId } from '../../reducers/tasks.reducer';
+
+import type { Task } from '../../types';
 
 const { app, process, Menu } = remote;
 
 type Props = {
+  selectedProjectId: ?string,
+  devServerTask: ?Task,
   createNewProjectStart: () => any,
   showImportExistingProjectPrompt: () => any,
-  clearConsole: () => any,
+  clearConsole: (task: Task) => any,
 };
 
-// TODO: Maybe I should store the git repo URL somewhere? Maybe it should
-// read from package.json?
-const baseRepoUrl = 'https://github.com/joshwcomeau/guppy';
+const baseRepoUrl = packageJson.repository.url.replace(/.git$/, '');
```

Comment:
Should this be ported into some sort of helper? We have a couple of references to Guppy documentation in the codebase and I can see `baseRepoUrl` being helpful to ensure links are consistent if the repo should ever change.

Reply:
Could even wrap the whole thing into a function to save a few more keystrokes, something like

```js
const fromRepo = path => packageJson.repository.url.replace(/.git$/, '') + path;
```

Reply:
So, my thing with `utils.js` is that it's meant to be a collection of utility functions, like `lodash`. Reading from a package.json feels too domain-specific (even if it's generalized to use any `package.json` and not specifically about Guppy, it still feels a bit weird to me).

I do agree that it'd be useful in other places though. I'm gonna put it in `read-from-disk.service.js`, since this service deals with a bunch of similar concerns.

Reply:
Hm, actually... `read-from-disk` mostly deals with the package.jsons of Guppy-created projects, not Guppy itself. Seems like it might get a bit confusing to bundle it in there.

I wish we had a `helpers.js` for this, although then we have 3 different places for these kinds of things, which feels like too many.

I'm going to create a `guppy-helpers.service.js`. I don't feel totally satisfied with that approach, so let me know if you have a better idea... but I think it works for now.